docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Using the Web UI
+description: Using the Teleport Web UI
```

Comment:
Same comment as `config.json`, the title and description should be more specific to the features of the Web UI being documented.